theseus_gui/src/helpers/profile.js

```diff
@@ -18,7 +18,7 @@ import { invoke } from '@tauri-apps/api/tauri'
 
 export async function create(name, gameVersion, modloader, loaderVersion, icon, noWatch) {
   //Trim string name to avoid "Unable to find directory"
-  name = name.trim();
+  name = name.trim()
```

Comment:
why was this semicolon removed -- make sure to format code according to modrinth's standards before pushing.